packages/builtins/src/collections/user/insert.ts

```
@@ -9,10 +9,79 @@ export const insert = async <
   payload: NoInfer<TInsertPayload>,
   context: Context<TDescription>,
 ) => {
+  const mutableProperties = context.config.security.mutableUserProperties
+  if(!context.token.authenticated){
+    return context.error(HTTPStatus.Unauthorized, {
+      code: ACError.AuthenticationError,
+    })
+  }
+  if(!mutableProperties){
```

This error return is misplaced. Setting `mutableUserProperties` to `undefined` is a developer problem, not a user one, so this clausule should throw instead of return an error. Also this property should be typed as required so this verification wouldn't be required at all.